pkg/plugin/restore_pvc_action_plugin.go

```diff
@@ -140,6 +142,31 @@ func (p *NewPVCRestoreItemAction) Execute(input *velero.RestoreItemActionExecute
 		return nil, errors.WithStack(err)
 	}
 
+	kubeClient, err := kubernetes.NewForConfig(restConfig)
```

Comment:
Would you please create a util function under pkg/plugin/util/ for the code snippet below? The `RetrieveStorageClassMapping` func below would be an example.

Reply:
Done